src/basic/tmpfile-util.c

```diff
@@ -45,6 +47,8 @@ int fopen_temporary(const char *path, FILE **_f, char **_temp_path) {
                 return -errno;
         }
 
+        (void) __fsetlocking(f, FSETLOCKING_BYCALLER);
```

Comment:
could you add a comment here explaining why this is OK? (i.e. the comment from the commit msg)

Reply:
Done.